macosx/TorrentTableView.mm

```diff
@@ -817,6 +817,9 @@ - (void)displayTorrentActionPopoverForEvent:(NSEvent*)event
     popover.behavior = NSPopoverBehaviorTransient;
     InfoOptionsViewController* infoViewController = [[InfoOptionsViewController alloc] init];
     popover.contentViewController = infoViewController;
+
+    //ignore custom height calculations in infoViewController and force height size
```

Comment:
We'd need to address this everywhere at some point, but comments should be with a space after the delimiter (`// `, `/* `, ...) to avoid two things:
- having the slash merged with a file path or URL `//~/Applications is for...`
- having the slash or star merged with a documentation comment `///`, `/**`, ...